testapp/src/main/java/com/rakuten/tech/mobile/testapp/helper/SingleExecution.kt

```diff
@@ -0,0 +1,18 @@
+package com.rakuten.tech.mobile.testapp.helper
+
+import android.os.SystemClock
+
+/**
+ * The simple controller which ensures only one method executed at a time.
+ */
+class SingleExecution {
```

Comment:
I'm not sure. Will the default `run` from Kotlin suffice? 
It has something like:
`contract {
        callsInPlace(block, InvocationKind.EXACTLY_ONCE)
    }`

Reply:
```contract { callsInPlace(block, InvocationKind.EXACTLY_ONCE) }```
-> could you please give me the resource document of this?

Reply:
Check out `Standard.kt`

Reply:
`contract` looks pretty cool. It's still experimental, but that should be okay since this is for the Demo App.

Reply:
The SingleExecution implementation here ensures only one type of method is going be executed at the specific time. i.e tap two different buttons on screen at the same time.
Using `InvocationKind.EXACTLY_ONCE` does not match the situation here. 

Perhaps, can you suggest the better name instead of `SingleExecution`?

Reply:
Got it. Thanks..!
I was looking at the kdoc in the file. How about `RaceController`. 
It is like a race between the functions. The one who reaches first wins the race.